packages/react/src/components/DataTable/DataTable.types.ts

```diff
@@ -256,14 +263,20 @@ export interface footerPropsType {
     headerId: number | string,
     index: number,
     keyword: string,
-  ) => number | string;
+  ) => any;
```

Comment:
Please rename `footerPropsType` to `FooterPropsType`
It's better to keep type names as Upper CamelCase

Reply:
fixed these